core/unit_test/CMakeLists.txt

```diff
@@ -44,8 +44,11 @@ if(Cabana_ENABLE_Pthread)
   foreach(_test AoSoA Slice DeepCopy Tuple Sort LinkedCellList NeighborList Parallel)
     add_executable(${_test}_test_Pthread Pthread/tst${_test}_Pthread.cpp unit_test_main.cpp)
     target_link_libraries(${_test}_test_Pthread cabanacore cabana_core_gtest)
-    add_test(NAME ${_test}_test_Pthread COMMAND ${_test}_test_Pthread --gtest_color=yes)
-    set_tests_properties(${_test}_test_Pthread PROPERTIES FAIL_REGULAR_EXPRESSION "${FAIL_RE}")
+    foreach(_thread 1 2)
+      add_test(NAME ${_test}_test_Pthread_${_thread} COMMAND
+        ${_test}_test_Pthread --gtest_color=yes --kokkos-threads=${_thread})
```

Comment:
Nice - yeah was going to tell you this was the way to add multiple pthreads to the kokkos runtime. We can do this instead of setting `OMP_NUM_THREADS` as well.

Reply:
On the other hand, with this, we could "simplify" the testing in Cmake by using a triple nested `foreach()` loop, do you want that?

Reply:
if you think that will be cleaner and easier to maintain go for it - otherwise I think this is fine the way it is

Reply:
Done!